modules/postgresql/metadata.display.yaml

```diff
@@ -40,9 +40,19 @@ spec:
         availability_type:
           name: availability_type
           title: Availability Type
+          alt_defaults:
+            - type: ALTERNATE_TYPE_DC
+              value: REGIONAL
         backup_configuration:
           name: backup_configuration
           title: Backup Configuration
+          alt_defaults:
+            - type: ALTERNATE_TYPE_DC
+              value:
```

Comment:
The value should be yaml equivalent of `{
  "enabled": true,
  "point_in_time_recovery_enabled": true
}`. I guest it should be like

```
value:
  enabled: true
  point_in_time_recovery_enabled: true
```